src/System.Text.Json/tests/Utf8JsonReaderTests.MultiSegment.cs

```diff
@@ -47,7 +47,7 @@ public static void TestJsonReaderUtf8SegmentSizeOne(bool compactData, TestCaseTy
             {
                 var utf8JsonReader = new Utf8JsonReader(sequence, isFinalBlock: true, default);
                 byte[] resultSequence = JsonTestHelper.ReaderLoop(dataUtf8.Length, out int length, ref utf8JsonReader);
-                string actualStrSequence = Encoding.UTF8.GetString(resultSequence.AsSpan(0, length));
+                string actualStrSequence = Encoding.UTF8.GetString(resultSequence, 0, length);
```

Comment:
These test changes will be reverted since we don't plan to build tests for netstandard.